docs/modules/ROOT/pages/references/decision-deployment-strategy.adoc

```diff
@@ -0,0 +1,192 @@
+= Decision: Deployment Strategy For Postgresql
```

Comment:
```suggestion
= Decision: Deployment Strategy For PostgreSQL
```

Please fix that in the whole document to be consistent with upstream / original writing.